### PR TITLE
Add .gitignore.

### PR DESCRIPTION
**Pull request by Tze1com**
Pretty self-explanatory.

### CODE REVIEW

**Comment by tlei123** [comment]
Where did the ignore rules come from?

**Comment by Tze1com** [comment]
> Where did the ignore rules come from?

create-react-app repo.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.idea/
+.vscode/
+node_modules/
+build
+.DS_Store
+*.tgz
+my-app*
```
Comment (tlei123) [inline .gitignore]: @Tze1com, how about adding tmp/ and temp/ dirs?

Comment (Tze1com) [reply .gitignore]: Dirs added after merging master into my branch (not rebase).

**Comment by Tze1com** [comment]
Holding off deleting my branch 'til all testing is complete.

**Pull request merged**